src/Storages/MergeTree/MergeTreeDataPartWriterCompact.cpp

```diff
@@ -365,8 +365,9 @@ void MergeTreeDataPartWriterCompact::addToChecksums(MergeTreeDataPartChecksums &
     {
         uncompressed_size += stream->hashing_buf.count();
         auto stream_hash = stream->hashing_buf.getHash();
+        transformEndianness<std::endian::little>(stream_hash);
```

Comment:
this doesn't work because `stream_hash` is type of `CityHash_v1_0_2::uint128` which is defined as `std::pair<uint64, uint64>` - there is no suitable specialization for `transformEndianness`

Reply:
It likely snuck in with a rebase, but all should be well now.